robot_ws/.rosinstall

```diff
@@ -1 +1,3 @@
 # Amazon ROS CloudServiceIntegration package dependencies
+- git: {local-name: src/deps/kinesisvideo-common, uri: 'https://github.com/aws-robotics/kinesisvideo-common.git', version: master}
+- git: {local-name: src/deps/kinesisvideo-ros2, uri: 'https://github.com/aws-robotics/kinesisvideo-ros2.git', version: master}
```

Comment:
This is needed until https://github.com/aws-robotics/kinesisvideo-ros2/pull/20 has been bloom released.